src/proto/streams/recv.rs

```diff
@@ -264,6 +265,21 @@ impl Recv {
                 // corresponding headers frame pushed to `stream.pending_recv`.
                 self.pending_accept.push(stream);
             }
+        } else {
+            // This is an informational response (1xx status code)
+            // Convert to response and store it for polling
+            let message = counts
+                .peer()
+                .convert_poll_message(pseudo, fields, stream_id)?;
+
+            tracing::trace!("Received informational response: {:?}", message);
```

Comment:
A message _could_ include sensitive information in the headers, so we try not to log the contents. If the trace itself is useful, perhaps just including the stream ID and the status code?